src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -42,12 +43,12 @@ public void ScanAssembly(Assembly asm)
 
     public IEnumerable<string> Generate(string[] pinvokeModules, string outputPath)
     {
-        var modules = new Dictionary<string, string>();
+        var modules = new SortedDictionary<string, string>(StringComparer.Ordinal);
```

Comment:
Do we need to canonize UTF before we insert it ?

Reply:
No, the ordinal sort is the same